release_util/tests/migrations/test_migrations/0004_fourth.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.2.3 on 2023-07-24 11:29
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
It was added due to https://docs.djangoproject.com/en/4.0/releases/4.0/#migrations-autodetector-changes

Reply:
Als this is test migration nothing to do with prod.